ingest.py

```diff
@@ -30,7 +30,7 @@
 # Map file extensions to document loaders and their arguments
 LOADER_MAPPING = {
     ".csv": (CSVLoader, {}),
-    # ".docx": (Docx2txtLoader, {}),
+    ".docx": (Docx2txtLoader, {}),
```

Comment:
If you want to do this, can't judge, you have to add the `Docx2txtLoader` class to the `from langchain.document_loaders import (...)` import above, or the script won't run.

Also, I think you should either comment out the other `.docx` entry right below, or change it to another filetype.

Reply:
Ah you are right. UnstructuredWordDocumentLoader does make more sense.

Although The load_dotenv() removal makes sense. There is an existing PR [#268 ](https://github.com/imartinez/privateGPT/pull/268)

Thanks @PulpCattel